scripts/install.js

```diff
@@ -24,10 +26,29 @@ var request = http.get(download_url, function(response) {
 	response.pipe(file);
 	file.on('finish', function() {
 		file.close(function(err){
-			var zip	= new AdmZip(zip_path);
-			zip.extractAllTo(dir_path, true);
+			// Verify file using hash make MITM harder
+			var fstream = fs.createReadStream(zip_path);
+			var hash = crypto.createHash('sha512');
+			hash.setEncoding('hex');
 			console.log('Download complete');
-			process.exit(0);
+			console.log('Starting integrity check...');
+
+			fstream.on('end', function() {
+				hash.end();
+				calculated_hash = hash.read();
+				if (zip_hash === calculated_hash){
+					var zip	= new AdmZip(zip_path);
+					zip.extractAllTo(dir_path, true);
+					console.log("Extraction complete")
+					process.exit(0);
+				}else{
+					console.error(`File verification failed:\nDownloaded file sha512: ${calculated_hash}`);
+					process.exit(-1);
```

Comment:
The important thing to avoid is the fact a file modified using MITM should be `deleted` before `exiting` :smile:.
I would suggest adding a `fs.unlink(...)` to delete the file and avoid the persistence on a computer if the `sha512` doesn't match.